### PR TITLE
Finally add a readme

### PR DESCRIPTION
**Pull request by maxmitti**
This was _so_ overdue …

### CODE REVIEW

**Review by cyberodactyl — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+Configure with CMake
+```PowerShell
+cmake . -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo
+```
+
+Open the resulting Visual Studio solution or simply compile with CMake as follows
+```PowerShell
+cmake --build build
+```
```
Comment (Fulgen301) [inline README.md:44]: Visual Studio can open CMake projects directly; anyone who doesn't want that already knows how to use the terminal.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+### Setup game folder
+The engine needs various files to run placed next to the engine binary.
+The cleanest way is to create and setup a game folder somewhere outside of the source code repository.
+
+If you want to use an existing game folder (e.g. obtained via the simple binary [Installation](#Installation)) use that folder instead and only do step 2.
+
+1. Create an empty folder at the desired location.
+The guide assumes the directory path to be `~/clonk`.
+1. Create a symbolic link to the binary in the folder
+	- Windows: `mklink /H ~/clonk/MyClonk.exe build/clonk.exe`
```
Comment (Fulgen301) [inline README.md:85]: `mklink /H` creates a hard link.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+	- Linux: `ln -s build/clonk ~/clonk/myclonk`
+	- macOS: `ln -s build/clonk.app ~/clonk/myclonk.app`
+1. Create symbolic links to [Graphics.c4g](planet/Graphics.c4g) and [System.c4g](planet/System.c4g). Without them, the engine won’t start.
+	- Windows
+	```PowerShell
+	mklink /J ~/clonk/Graphics.c4g planet/Graphics.c4g
+	mklink /J ~/clonk/System.c4g planet/System.c4g
```
Comment (Fulgen301) [inline README.md:92]: PowerShell actually doesn't know `mklink` - this should be
```suggestion
	New-Item -ItemType Junction -Path ~/clonk/Graphics.c4g -Target planet/Graphics.c4g
	New-Item -ItemType Junction -Path ~/clonk/System.c4g -Target planet/System.c4g
```

**Review by Fulgen301 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (maxmitti) [reply README.md:85]: I am confused about the different links there are in Windows.
Are you actively using some? If so, which mode should it be?

File: README.md (hunk 3 above)
Comment (maxmitti) [reply README.md:92]: Good point. Do you know how to link multiple items with wildcard? (Regarding content *.c4?)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+### Setup game folder
+The engine needs various files to run placed next to the engine binary.
+The cleanest way is to create and setup a game folder somewhere outside of the source code repository.
+
+If you want to use an existing game folder (e.g. obtained via the simple binary [Installation](#Installation)) use that folder instead and only do step 2.
+
+1. Create an empty folder at the desired location.
+The guide assumes the directory path to be `~/clonk`.
+1. Create a symbolic link to the binary in the folder
+	- Windows: `New-Item -ItemType Junction -Path ~/clonk/MyClonk.exe -Target build/clonk.exe`
```
Comment (maxmitti) [inline README.md:75]: @Fulgen301, is this correct?

**Review by Fulgen301 — APPROVED** [review]

**Pull request merged**